operations/installation.rst

```diff
@@ -33,15 +45,19 @@ a ``$basearch`` variable. For example:
        charm: slurmd
        num_units: 1
        series: focal
-       custom-slurm-repo: ppa:omnivector/osd-testing
+       custom-slurm-repo: ppa:omnivector/osd
      ...
 
+Another possibility is to supply the repository URL in the command line when
+deploying the units:
 
-This also allows one to specify different repositories for testing new Slurm
-versions, or upgrading only specific charms.
+.. code-block:: bash
+
+   $ juju deploy slurmd --series centos7 --config custom-slurm-repo="https://omnivector-solutions.github.io/repo/centos7/stable/$basearch"
```

Comment:
won't the terminal try to find the variable `basearch` in this case?

Reply:
Good catch. I changed the example to address it.